libraries/libfc/test/crypto/test_webauthn.cpp

```diff
@@ -52,7 +52,7 @@ BOOST_AUTO_TEST_CASE(good) try {
 
    memcpy(auth_data.data(), origin_hash.data(), sizeof(origin_hash));
 
-   BOOST_CHECK_EQUAL(wa_pub, make_webauthn_sig(priv, auth_data, json).recover(d, true));
+   BOOST_CHECK_EQUAL(wa_pub, make_webauthn_sig(priv, auth_data, json).recover(d, check_canonical_t::yes));
```

Comment:
Since `webauthn` are not canonical, seems like we should change all of these to pass `no`.